assets/shop/order_show_entrypoint.js

```diff
@@ -2,5 +2,7 @@ import './js/retry_payment';
 import {CardForm} from "./js/card_form";
 
 document.addEventListener('DOMContentLoaded', () => {
-  new CardForm('[name="sylius_checkout_select_payment"]');
+  if (document.querySelector('[name="sylius_checkout_select_payment"]')) {
```

Comment:
In my PR I am also checking for input fields (related to card) to exist cause, otherwise it will also run when f.e. Visa Mobile or BLIK payment asks for input. Name of form is the same 🤷 

My change for ref. https://github.com/CommerceWeavers/SyliusTpayPlugin/pull/85/files#diff-5870a8fc565aeb435186c7ad9e52b82b5a5776ef73b4ec76c7bf8d33d4747a87R8

Reply:
We'll solve it while processing conflicts, depending which PR will be merged first.